src/Fixer/StringNotation/EscapeImplicitBackslashesFixer.php

```diff
@@ -0,0 +1,156 @@
+<?php
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\Fixer\StringNotation;
+
+use PhpCsFixer\AbstractFixer;
+use PhpCsFixer\Fixer\ConfigurationDefinitionFixerInterface;
+use PhpCsFixer\FixerConfiguration\FixerConfigurationResolver;
+use PhpCsFixer\FixerConfiguration\FixerOptionBuilder;
+use PhpCsFixer\FixerDefinition\CodeSample;
+use PhpCsFixer\FixerDefinition\FixerDefinition;
+use PhpCsFixer\Tokenizer\Token;
+use PhpCsFixer\Tokenizer\Tokens;
+
+/**
+ * @author Filippo Tessarotto <zoeslam@gmail.com>
+ */
+final class EscapeImplicitBackslashesFixer extends AbstractFixer implements ConfigurationDefinitionFixerInterface
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function getDefinition()
+    {
+        $codeSamble = <<<'EOF'
+<?php
+
+$singleQuoted = 'String with \" and My\Prefix\\';
+
+$doubleQuoted = "Interpret my \n but not my \a";
+
+$hereDoc = <<<HEREDOC
+Interpret my \100 but not my \999
+HEREDOC;
+
+EOF;
+
+        return new FixerDefinition(
```

Comment:
https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/3155#issuecomment-336856953 is still awaiting
When John Doe will look at rule, he will have no clue why he shall follow it.

Reply:
Tell me if it's ok now.

Reply:
In PR, to advocate the feature, you had to put a lot of description.
Yet, in `FixerDefinition`, you use only short `summary` parameter (1st param). You know that it also has 3rd param for extended, detailed `description` ? :)